django/test/selenium.py

```diff
@@ -133,3 +137,86 @@ def disable_implicit_wait(self):
             yield
         finally:
             self.selenium.implicitly_wait(self.implicit_wait)
+
+
+def screenshot_cases(method_names):
+    if isinstance(method_names, str):
+        method_names = method_names.split(",")
+
+    def wrapper(func):
+        func._screenshot_cases = method_names
+        return func
+
+    return wrapper
+
+
+@tag("screenshot")
+class ScreenshotTestCase(SeleniumTestCase):
+    screenshots = False
+
+    @classmethod
+    def __init_subclass__(cls, **kwargs):
```

Comment:
@adamchainz I took a lot of inspiration from your package [unittest-parametrize](https://github.com/adamchainz/unittest-parametrize) here :heart: 
Feel free to add your thoughts on how best to structure this :+1:

Reply:
Also @ngnpope if you want to look at the PR again, I approach has changed quite a bit to accommodate different requirements and would be good to get some feedback :star:

Reply:
Glad to inspire!